### PR TITLE
add functions to check if any validation errors

### PR DESCRIPTION
**Pull request by lmiller207**

### CODE REVIEW

File: src/Controller.js [hunk 1]
```diff
@@ -41,6 +41,10 @@ class Controller {
     return this.itemStore.isDirty();
   }
 
+  errors() {
```
Comment (sfhardman) [inline src/Controller.js]: hasErrors to be consistent and indicate this is a boolean, not returning the set of errors

**Comment by sfhardman** [comment]
Looks fine, other than the one comment on naming

File: src/Controller.js (hunk 1 above)
Comment (lmiller207) [reply src/Controller.js]: I did think that too, I was trying to keep it consistent with the function above it which goes from isDirty() in the itemStore to dirty() in the controller
Should I change that one also?

Comment (sfhardman) [reply src/Controller.js]: Oops.  No, I think just change your one.  I'm doing some refactoring at the moment and will try to remember to tidy dirty()

**Pull request merged**